packages/account-v2/src/components/CountrySelector/CountrySelector.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useResidenceList } from '@deriv/api';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Dropdown } from '@deriv-com/ui';
+
+type TCountrySelector = {
+    errorMessage?: string;
```

Comment:
Lets make this as a reactNode since we may pass a component like <Trans or <Localize in the future